ldap/servers/plugins/uiduniq/uid.c

```diff
@@ -770,13 +770,13 @@ search_one_berval(Slapi_DN *baseDN, const char **attrNames, const struct berval
  *
  * Return:
  *   LDAP_SUCCESS - no matches, or the attribute matches the
- *     target dn.
+ *     destination (target) dn.
```

Comment:
Sorry but I  am confused. With a MODRDN we have a  "sourceDN -> destinationDN". In my mind "target" is meaning SLAPI_<op>_TARGET_SDN, that is "sourceSDN". IIUC target is souce not destination.